### PR TITLE
Add lihan li

### PR DESCRIPTION
**Pull request by lihanli**

### CODE REVIEW

**Comment by greggersh** [comment]
👍

File: _team/li-lihan.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title: Lihan Li
+position: Senior Software Engineer
+github_username: lihanli
+photo: /assets/img/content/li-lihan.jpg
+links:
+  - title: lihanli.com
+    url: http://lihanli.com/
+---
+
+Lihan is an engineer engineering emazing ebsites.
```
Comment (benkutil) [inline _team/li-lihan.md]: @lihanli you wanted this alliteration, correct?

`engineer engineering emazing ebsites.`

Comment (lihanli) [inline _team/li-lihan.md]: @benkutil yea

Comment (benkutil) [inline _team/li-lihan.md]: 👍

**Comment by benkutil** [comment]
👍

**Pull request merged**